src/jit/Backend.cpp

```diff
@@ -84,10 +84,16 @@ void JITArg::set(Operand* operand)
         return;
     }
 
-    this->arg = SLJIT_IMM;
-
     Instruction* instr = operand->item->asInstruction();
 
+    if (instr->opcode() == ByteCode::Const128Opcode) {
+        this->arg = SLJIT_MEM0();
+        this->argw = (sljit_sw)reinterpret_cast<Const128*>(instr->byteCode())->value();
```

Comment:
There is a simd specific loader, is there an issue with it?

Reply:
Yes. It doesn't work for this case. This is for unaligned memory.

Reply:
I want to know whether this code needs to be here, or can be in simd specific part of the code. It looks out of context.